### ISSUE TITLE
Bug: Unix format (includes Macintosh) class list CSV data will not propogate to DB

### ISSUE THREAD

**Issue by pbailie** [RCOS-Grading-Server/HWserver#241]
Currently, the class-list submit code splits the whole CSV into rows by looking for a ```\r``` delimiting character.  This only works in Windows formatted CSV files -- every row is terminated with ```\r\n```.  Unix formatted CSVs (including those created with a Macintosh) have rows terminated with only ```\n```.  Therefore, a entire CSV file created/converted on Macintosh is interpreted as a one humongous single row.

When uploading a Unix formatted CSV, no error message is reported to the Instructor/TA, but the students table in DB is not filled.  No exception is thrown.

I will correct this bug while working on issue #239.

- [x] Correct class list submit code to be compatible with Unix Format (terminated with ```\n```) CSV files.

**Comment by pbailie** [comment]
I am aware that looking for ```\n``` as a row-delimiting character is not a viable solution.

**Comment by MasterOdin** [comment]
Yeah, that was dumb of me to do, whoops.
Here's how I should have done it:
```
$lines = file($_FILES['classlist']['tmp_name'], FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
foreach ($lines as $line) {
    // rest of the code
}
```

**Comment by pbailie** [comment]
Looks good.  Thanks!  I'll get this implemented as I work on #239.
